api/app/applicationModel/api.py

```diff
@@ -31,15 +35,19 @@ class ApplicationFormAPI(ApplicationFormMixin, restful.Resource):
         'placeholder': fields.String,
         'validation_regex': fields.String,
         'validation_text': fields.String,
-        'is_required': fields.Boolean
-    } 
+        'is_required': fields.Boolean,
+        'depends_on_question_id': fields.Integer,
+        'show_for_values': fields.Nested(show_for_values_fields)
+    }
 
     section_fields = {
         'id': fields.Integer,
         'name': fields.String,
         'description': fields.String,
-        'order': fields.Integer,  
-        'questions': fields.List(fields.Nested(question_fields))   
+        'order': fields.Integer,
+        'questions': fields.List(fields.Nested(question_fields)),
+        'depends_on_question_id': fields.Integer,
+        'show_for_values': fields.Nested(show_for_values_fields)
```

Comment:
So I think it will need to be `fields.List(fields.Nested(show_for_values_fields)` and then

```
show_for_values_fields = {
  'value': fields.String
}
```

I'm basing that off how the `question_fields` and `option_fields` are setup in this same file except we just going to have a `value` and no `label`

Reply:
Thank you for the quick feedback. Will do the changes now.